src/Query/Aggregation/AbstractAggregation.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query\Aggregation;
+
+use TBolier\RethinkQL\Query\AbstractQuery;
+
+abstract class AbstractAggregation extends AbstractQuery implements AggregationInterface
```

Comment:
We do not need an abstract if it is empty

Reply:
It's not empty anymore